### PR TITLE
Feature/v3 header

### PR DESCRIPTION
**Pull request by jmzwar**
Screen Shot 2023-01-22 at 8 26 52 pm
Screen Shot 2023-01-22 at 8 26 58 pm

### CODE REVIEW

File: v3/components/icons/ChevronDown/ChevronDown.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Icon, IconProps } from '@chakra-ui/react';
```
Comment (bachstatter) [inline v3/components/icons/ChevronDown/ChevronDown.tsx:1]: I think chakra icons might have the chevrons

File: v3/ui/src/components/NetworkController.tsx [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+  EthereumIcon,
+  OptimismIcon,
+  FailedIcon,
+  ChevronUp,
+  ChevronDown,
+  WalletIcon,
+} from '@snx-v3/icons';
 import { useAccount } from '@snx-v3/useBlockchain';
 
+import { switchNetwork } from '@wagmi/core';
```
Comment (bachstatter) [inline v3/ui/src/components/NetworkController.tsx:13]: https://wagmi.sh/react/hooks/useSwitchNetwork I think we should use this instead, then we dont have to add wagmi/core

File: v3/ui/package.json [hunk 3]
```diff
@@ -54,6 +54,7 @@
     "@synthetixio/wei": "workspace:*",
     "@tanstack/react-query-devtools": "^4.3.5",
     "@wagmi/chains": "^0.1.5",
+    "@wagmi/core": "latest",
```
Comment (bachstatter) [inline v3/ui/package.json:57]: This should not be latest. But also I dont think we need it. See comment about swtich netowrk

**Comment by bachstatter** [comment]
It should also display the account when you have a position, so make sure you do some testing with an account too

**Review by bachstatter — APPROVED** [review]

File: v3/ui/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmzwar) [reply v3/ui/package.json]: removed

File: v3/ui/src/components/NetworkController.tsx (hunk 2 above)
Comment (jmzwar) [reply v3/ui/src/components/NetworkController.tsx]: Ta - done

**Pull request merged**